### DISCUSSION TITLE
Cannot scan GitHub 2FA QR code when using GitHub dark theme

### DISCUSSION THREAD

**Discussion by steffan-westcott** [community/community#27814] [Other Feature Feedback, Questions, & Ideas]
This is a bug report for an unfortunate interaction in GitHub's web interface which makes QR codes unusable when a GitHub "dark theme" is used. Using a GitHub "light theme" fixed the issue.

Steps to reproduce:
- Log into GitHub account with no 2FA enabled
- Navigate to page Settings > Appearance and select "Dark default" as the theme
- Navigate to page Settings > Password & authentication and click button "Enable two-factor authentication"
- Select "Set up using an app" and click continue
- Attempt to scan the presented QR code using any authenticator app

Expected result:
- Authenticator app scans the QR code and proceeds to display a 6-digit code

Actual result:
- QR code cannot be scanned by any app

Workaround:
- Select "Light default" as a theme before setting up 2FA. The QR code can then be scanned easily.

**Comment by NikitaVasileusky** [comment]
The same issue, guys. Registered new account, tried to add two-factor, qr code did not worked for me.  Tried on 3 phones : Samsung s10e (microsoft authenticatr + authy), iphone 11, samsung s7 - no one recognized QR. I also copied qr code and pasted to url (base64 svg?), opened in firefox and tried again - without any chances. So, fix with changing from black to white scheme is a key to resolve. Please, fix it :)

**Comment by joshuawalker** [comment]
Hey @steffan-westcott and @NikitaVasileusky—thanks for letting us know! We'll get this fixed up.

**Reply by kjhyhy6765** [reply]
four years later and I STILL HAVE THE SAME ISSUE! Thank you Steffan.

**Reply by Ary3ndra** [reply]
even i'm facing the same issue
